code/game/dna/genes/vg_powers.dm

```diff
@@ -242,6 +252,83 @@
 		for(var/mob/dead/observer/G in GLOB.player_list)
 			G.show_message("<i>Telepathic message from <b>[user]</b> ([ghost_follow_link(user, ghost=G)]) to <b>[target]</b> ([ghost_follow_link(target, ghost=G)]): [say]</i>")
 
+/obj/effect/proc_holder/spell/targeted/mindscan
+	name = "Scan Mind"
+	desc = "Offer people a chance to share their thoughts!"
+	charge_max = 0
+	clothes_req = 0
+	stat_allowed = 0
+	invocation_type = "none"
+	range = -2
+	selection_type = "range"
+	action_icon_state = "genetic_mindscan"
+	var/list/available_targets = list()
```

Comment:
may want to `Cut` this list if the spell is ever removed or something.

Reply:
Not depicted automatically, but `Cut` was added to the spell's `Destroy()` in the most recent commit.